protected/config/main.php

```diff
@@ -8,7 +8,7 @@
 // CWebApplication properties can be configured here.
 return array(
     'basePath' => dirname(__FILE__).DIRECTORY_SEPARATOR.'..',
-    'name'     => 'My Web Application',
+    'name'     => 'Onboarding Exercises',
```

Comment:
Hi JL! So far your work is good. I will summarize here my comments.
Continue:
1) Typecasting to prevent PHP's type juggling.
2) Using strict comparison `===`
3) Following PSR-1 guidelines.

For improvement,
Try:
1) Removing boilerplate code
2) Limiting loops within loops

Study:
1) PSR-1 and PSR-2 guidelines
2) Using namespaces